tests/visco_plastic_2_phases_compositions_lookups.prm

```diff
@@ -0,0 +1,122 @@
+# Copy of visco_plastic_phases.prm, but include multiple
+# compositions and have different numbers of phase transitions
+# for different compositions (including 0).
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 1000
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = no Advection, no Stokes
+set Max nonlinear iterations               = 1
+set Output directory                       = visco_plastic_lookup
+set Timing output frequency                = 1
+
+# Model geometry (100x100 km, 10 km spacing)
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set X extent      = 100e3
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 2
+  set Time steps between mesh refinement = 0
+end
+
+# Velocity on boundaries characterized by functions
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = bottom y: function, top y: function, left x: function, right x: function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = m=0.0005, year=1
+    set Function expression = if (x<50e3 , -1*m/year, 1*m/year); if (y<50e3 , 1*m/year, -1*m/year);
+  end
+end
+
+# Temperature boundary and initial conditions
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top, left, right
+  set List of model names = box
+  subsection Box
+    set Bottom temperature = 273
+    set Left temperature   = 273
+    set Right temperature  = 273
+    set Top temperature    = 273
+  end
+end
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 273
+  end
+end
+
+subsection Compositional fields
+  set Number of fields = 2
+  # background is assumed to be lithosphere
+  set Names of fields  = upper_crust, lower_crust
+end
+
+subsection Initial composition model
+  set Model name = function
+
+  subsection Function
+    set Function expression = (y > 80000) ? 1.0 : 0.0; \
+                              (y > 50000) ? ((y <= 80000) ? 1.0 : 0.0) : 0.0
+  end
+end
+
+subsection Material model
+  set Model name = visco plastic
+  subsection Visco Plastic
+    set Data directory                   = $ASPECT_SOURCE_DIR/data/material-model/steinberger/test-steinberger-compressible/
+    set Material file names              = constant_material_small.txt, constant_material_2_small.txt, constant_material_small.txt
```

Comment:
Can you add a comment regarding what is contained within these files. It would also be helpful to add this description at the beginning of the PRM file.

Reply:
Good idea. Added a brief description to both of the .prm files.